README.md

```diff
@@ -32,7 +34,7 @@ To get the metadata in a single record, use the following type of URL:
 https://inspirehep.net/api/{identifier-type}/{identifier-value}
 ```
 
-Two main categories of identifiers are supported.
+Two main categories of record identifiers are supported.
 
 ### Internal identifiers
```

Comment:
@tsgit commented:
> I wouldn't call "literature" an "identifier", rather a "document type" or
"resource type"

I don't, `literature` is an `identifier-type` which is part of the identifier in https://github.com/inspirehep/rest-api-doc/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R34 (the fact we have a global sequential pool of record IDs in an implementation detail).